main.cpp

```diff
@@ -10,17 +10,16 @@
 #include <vector>
 
 #ifdef _WIN32
-#include <ctime>
+	#include <ctime>
 #else
-#include <sys/time.h>
-#include <unistd.h>
-#include <sys/stat.h>
+	#include <sys/time.h>
+	#include <unistd.h>
+	#include <sys/stat.h>
 #endif
+
 #include <cstdlib>
 #include <cstdarg>
 
-//#define DEBUG
```

Comment:
Also keep it, please!

Reply:
Sure, I will deal with the DEBUG related code shortly.  Thanks!